parquet/src/main/java/org/apache/iceberg/parquet/ParquetUtil.java

```diff
@@ -99,6 +105,9 @@ public static Metrics footerMetrics(ParquetMetadata metadata, Stream<FieldMetric
     MessageType parquetTypeWithIds = getParquetTypeWithIds(metadata, nameMapping);
     Schema fileSchema = ParquetSchemaUtil.convertAndPrune(parquetTypeWithIds);
 
+    Map<Integer, FieldMetrics> fieldMetricsMap = fieldMetrics.collect(
```

Comment:
`FieldMetrics` is parameterized, right? I think this should be `FieldMetrics<?>`.

Reply:
I will modify the ones for Parquet and ORC, and leave the Avro related ones as is, since we are going to modify them in the Avro metrics support PR anyway. Let me know if it works!